src/dags/vergunningen.py

```diff
@@ -8,11 +8,9 @@
 from common import (
     OTAP_ENVIRONMENT,
```

Comment:
Why is the OTAP_ENVIRONMENT needed here? In the most dags this is deleted.

Reply:
Same reason as for riool, so basically to get the correct folder in objectstore (acceptance, production and such)